src/App.js

```diff
@@ -1,22 +1,20 @@
 import React from 'react';
 import './App.css';
 import { Routes, Route } from 'react-router-dom';
-// import { Provider } from 'react-redux';
+import { Provider } from 'react-redux';
 import Navigation from './components/Navigation';
 import Categories from './components/Categories';
 import BooksPage from './components/BooksPage';
-// import { store } from './redux/books/books';
+import store from './redux/configureStore';
 
 const App = () => (
-  // <Provider store={store}>
-  <>
+  <Provider store={store}>
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/use_redux_in_react_components.md) for this project and best practice, " `Import Redux Provider and your store in the main component of your React app.` ", The main React component (`App.js`) should be wrapped in Provider and assign store. Something like so: 
```
    import store from './redux/configureStore';

    ReactDOM.render(
    <Provider store={store}>
        <App />
    </Provider>,
    document.getElementById('root')
    )
```

Reply:
Okay, I understand